etc/requirements.txt

```diff
@@ -20,7 +20,7 @@ six==1.6.1
 # For fetching remote data
 requests==2.5.1
 
-Cython==0.20.1
+Cython>=0.20.1
```

Comment:
We use specific versions in requirements.txt, not >=, so this should not be changed.